resources/views/newsletter/subscribe.blade.php

```diff
@@ -1,25 +1,62 @@
+            <svg class="absolute inset-0 h-full w-full" preserveAspectRatio="xMidYMid slice"
+              xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 1463 360">
+              <path class="text-indigo-500 text-opacity-40" fill="currentColor"
+                d="M-82.673 72l1761.849 472.086-134.327 501.315-1761.85-472.086z" />
+              <path class="text-indigo-700 text-opacity-40" fill="currentColor"
+                d="M-217.088 544.086L1544.761 72l134.327 501.316-1761.849 472.086z" />
+            </svg>
+          </div>
+          <div class="relative">
+            <div class="sm:text-center">
+              <h2 class="text-3xl font-extrabold text-white tracking-tight sm:text-4xl">What would you like to subscribe?
+              </h2>
+              <p class="mt-6 mx-auto max-w-2xl text-lg text-indigo-200">Please, feel free to choose your preference!</p>
             </div>
-        @endif
-        <form action="{{route("newsletter.update")}}" method="post">
-            @csrf
-            <input type="email" hidden="hidden"  name="email" id="email" value="{{ $subscriber->email }}">
-            <label for="Notification_type">What would you like to subscribe?</label>
-            <div>
-                <label for="News">New News
-                    <input type="checkbox" name="type[]" id="News" checked="checked" value={{AvailableNewsletter::News->value}}></label>
-                <label for="News">New Meetups
-                    <input type="checkbox" name="type[]" id="Meetups" value={{AvailableNewsletter::Meetups->value}}></label>
-                <x-input-error for="type"/>
-            </div>
-            <input type="submit" name="Subscribe" value="Subscribe">
-        </form>
+            <form action="{{ route('newsletter.update') }}" class="mt-12 sm:mx-auto sm:max-w-lg" method="post">
+              @csrf
+              <input type="email" hidden="hidden" name="email" id="email" value="{{ $subscriber->email }}">
+              <fieldset class="border-t border-b border-indigo-200">
+                <legend class="sr-only">Notifications</legend>
+                <div class="divide-y divide-gray-200">
+                  <div class="relative flex items-start py-4">
+                    <div class="min-w-0 flex-1 text-sm">
+                      <label for="meetups" class="font-medium text-white">New Meetups</label>
+                      <p id="mettups-description" class="text-indigo-200">Get notified about new meetups that might
+                        interest you</p>
+                    </div>
+                    <div class="ml-3 flex items-center h-5">
+                      <input id="meetups" name="type[]" checked type="checkbox" value={{ AvailableNewsletter::Meetups->value }}
+                        class="h-[18px] w-[18px] text-indigo-600 border-gray-300 rounded default:border-gray-300 checked:border-gray-300 checked:border-2">
```

Comment:
Is really a good idea to write like this? `h-[18px] w-[18px]`

Reply:
Default tailwind widths and heights are not very accurate, so i picked this method. I just read about this, and it's nothing wrong with using arbitrary values, when styling is not repeatable.

Reply:
I changed the buttons. I think they are more visible now. As to checkboxes. They are very difficult to style. How they look now is the best result I managed to accomplish with @Majkulnn. Alternatively we can fix them later.

As for subscription. We decided with you that newsletter should be available for logged and and unlogged users. That's why there are subscribe and unsubscribe buttons.